portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -0,0 +1,27 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/form-handler")
+public class FormHandlerServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    // Get the value entered in the form.
+    String textValue = request.getParameter("text-input");
+
+    // Print the value so you can see it in the server logs.
+    System.out.println("You submitted: " + textValue);
+
+    //Redirect user back to portfolio page
+    response.sendRedirect("http://sliu-sps-summer22.appspot.com ");
+
+    // Write the value to the response so the user can see it.
+    // response.getWriter().println("You submitted: " + textValue);
```

Comment:
Did you mean to leave this commented out?

Reply:
Yes, I was testing whether the servlet should redirect back to the portfolio page or stay on a text page after the data is submitted. I'm planning on redirecting the user back to the portfolio page.